tests/album-create.test.js

```diff
@@ -0,0 +1,32 @@
+const { expect } = require('chai')
+const request = require('supertest')
+const getDb = require('../src/services/db')
+const app = require('../src/app')
+
+describe('create album', () => {
+  let db
+  //  establishing connection to db before each test
+  beforeEach(async () => (db = await getDb()))
+  // uses db.query to delete all the artists in the Artist table then closes the connection to the database
+  afterEach(async () => {
+    await db.query('DELETE FROM Album')
+    await db.close()
+  })
+
+  describe('/artist/:artistId/album', () => {
+    describe('POST', () => {
+      it('creates a new album associated to an artist in the database', async () => {
+        const res = await request(app).post('/artist/:artistId/album').send({
```

Comment:
You will need to create an artist in the database and use their ID number as part of this query string, this route cannot be found because of this and is causing the test to fail. You could add an artist as you in `tests/artist-delete.js` and this should fix the problem